Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -251,72 +245,50 @@ def get_certificate_details_command(client: Client, args: dict[str, Any]) -> Com
         client (Client): A Venafi client.
         args (dict): The arguments for the command passed to the request.
     Returns:
-        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+        A CommandResult object with an outputs, raw response and readable table, in case of a successful action.
     """
 
-    outputs: dict[str, Any] = {}
-    guid = args.get('guid', '')
-    response = client._get_certificate_details(guid)
-    if response:
-        outputs = response
-
-    human_readable = []
-    certificate_details = get_human_readable_object(outputs)
-    human_readable.append(certificate_details)
-
-    markdown_table = tableToMarkdown('Venafi certificate details', human_readable)
+    guid: str = args.get('guid', '')
```

Comment:
As said before, if the guid is required we don't have to assign a default value.